ffv1.md

```diff
@@ -731,10 +731,32 @@ Parameters( ) {                                               |
     colorspace_type                                           | ur
     if (version >= 1)                                         |
         bits_per_raw_sample                                   | ur
-    chroma_planes                                             | br
-    log2_h_chroma_subsample                                   | ur
-    log2_v_chroma_subsample                                   | ur
-    extra_plane                                               | br
+    if (colorspace_type < 2) {                                |
+        chroma_planes                                         | br
+        log2_h_chroma_subsample                               | ur
+        log2_v_chroma_subsample                               | ur
+        extra_plane                                           | br
+    } else {                                                  |
+        plane_count                                           | ur
+        per_plane_metadata_count                              | ur
+        for( i = 0; i < plane_count; i++ ) {                  |
+            plane_type                                        | ur
+            center_frequency                                  | ur
+            frequency_response ( center_frequency )           |
+            color_transfer_characteristic                     | ur
```

Comment:
`color_transfer_characteristic ` is common to YUV (+colour_primaries, matrix_coefficients), shouldn't it be in a color info dedicated part e.g. at the end of the configuration box?

Reply:
hmm, the idea was that we need something to go from physical energy to binary values and that often is not linear. Maybe color_transfer_characteristic is a poor term for this.

Reply:
`transfer_characteristic` wording would be misleading as the name is already used elsewhere (prefix 'color' would not be enough for differentiation IMO), so in that case it should be another name.

Reply:
* `color_translation`?
  * `color_conversion`?
  * `color_interpretation`?

possibly add `_characteristic` to any of those?